internal/dto/media.go

```diff
@@ -36,3 +28,19 @@ type UpdateMediaResponse struct {
 type DeleteMediaRequest struct {
 	MediaID uint `params:"mediaId"`
 }
+
+func ToMediaResponse(media model.Media) MediaResponse {
+	return MediaResponse{
+		ID:          media.ID,
+		PictureURL:  media.PictureURL,
+		Description: media.Description,
+	}
+}
+
+func ToMediaResponses(media []model.Media) []MediaResponse {
+	responses := make([]MediaResponse, 0)
```

Comment:
pre-alloc mem ได้เลย 

```suggestion
	responses := make([]MediaResponse, 0, len(media))
```